### PR TITLE
Initial conditions and volume boost

### PR DESCRIPTION
**Pull request by sajattack**

### CODE REVIEW

File: reDIP-SID/i2s_init_data.hex
Comment (sajattack) [inline reDIP-SID/i2s_init_data.hex:61]: I think the DAP will just be in passthrough mode with the GEQ stuff below commented out, but you might want to double check before merging

Comment (sajattack) [reply reDIP-SID/i2s_init_data.hex:61]: If it doesn't work, we'll have to change the routing back.

Comment (bit-hack) [reply reDIP-SID/i2s_init_data.hex:61]: I'll merge first and can commit after if its not as expected. :+1:

**Review by bit-hack — APPROVED** [review]

**Pull request merged**
